substra/sdk/backends/local/compute/worker.py

```diff
@@ -20,7 +20,7 @@
 
 from substra.sdk import schemas, fs, models
 from substra.sdk.backends.local import dal
-from substra.sdk.backends.local.compute import spawner
+from substra.sdk.backends.local.compute.spawner import DockerSpawner
 
 LOCAL_DIR = pathlib.Path.cwd() / "local-worker"
```

Comment:
This variable is used by the local backend to define the chainkeys dir. Since the local backend now creates the `Path.cwd() / "local-worker"` dir I don't think it's necessary to define this variable here.

I'm sorry this is probably not clear, but basically I'm not comfortable with Local importing a constant from worker.py just to pass it back to the Worker instance from this same worker.py file.

Reply:
If I understand well, we could definie `LOCAL_DIR` directly in one of the parent module, that is to say `local/backend.py` ?

Reply:
If we do so, we could also remove the `self._local_worker_dir` property. 

Or go the otherway around, keep the `self._local_worker_dir` property and remove the constant.

Reply:
You are right, it is convoluted. Changed it to defining it in the init of the backend, and it gives it to the worker, which gives it to the spawner. Also put back the `spawner.get` method, no reason to remove it.